monai/apps/mmars/mmars.py

```diff
@@ -285,7 +285,7 @@ def load_from_mmar(
     else:
         model_inst = model_cls()
     if pretrained:
-        model_inst.load_state_dict(model_dict.get(model_key, model_dict))
+        model_inst.load_state_dict(model_dict.get(model_key, model_dict), strict=False)
```

Comment:
if `strict=False` and put a wrong model, maybe it doesn't give any error?

Thanks.